.github/workflows/ci.yml

```diff
@@ -146,8 +146,10 @@ jobs:
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v5.25.0
       - name: "Build application"
         run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
+      - name: "Build docker images"
+        run: bash ./scripts/ci/buildDockerImages.sh
       - name: "Run e2e tests"
-        run: mvn ${{ env.MAVEN_CLI_OPTS }} verify -pl 'prediction-applier-extension,e2e-test' -DskipUnitTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
+        run: mvn ${{ env.MAVEN_CLI_OPTS }} verify -pl 'prediction-applier-extension,e2e-test' -am -DskipUnitTests -DskipIntegrationTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
```

Comment:
hah, you basically reverted my last change😂 I had to remove `-DskipIntegrationTests` because it was skipping `PredictionApplierExtensionIntegrationTest` and then I had to remove `-am` option because it was running integration tests from all modules. I've added 
```
- name: "Build application"
        run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -Dalfresco-platform.version=${{ matrix.repoVersion }}
```
step above to build whole app (I think it is the reason we use `-am` option)

Alternatively we could rename `PredictionApplierExtensionIntegrationTest` to `PredictionApplierExtensionE2ETest`(even thought it is actually integration test) and revert my changes

Reply:
The test: `PredictionApplierExtensionIntegrationTest` is an integration test and shouldn't run with E2E tests. This test: https://hyland.atlassian.net/browse/ACS-7742 should run with E2Es and verify if `hxi:` model is installed and works properly. Option `-am` only builds dependent modules, not all of them, but building all will be useful for next step - building docker images. I had to add back this `-am` option, because changes in `common-test` wasn't seen from `e2e-test` module and resulted in failing pipeline